.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: samples
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+jobs:
+  build:
+    name: Build and push docker images
+    runs-on: ubuntu-latest
+    env:
+      SAMPLE_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Parse release version and set REL_VERSION
+        run: python ./.github/scripts/get_release_version.py
+      - name: Build docker image
+        run: |
+          SAMPLE_LIST=(2.hello-kubernetes 3.distributed-calculator 4.pub-sub 5.bindings)
```

Comment:
what about sample 6?

Reply:
We have not upload sample6's docker images yet. I heard from @yaron2 that document describes the instructions starting from docker image builds. so he said we do not need to push the sample6. maybe we need later :) but not now.